common_ci_utils/random_utils.py

```diff
@@ -19,7 +19,7 @@ def generate_random_hex(length=20):
         str: A random hexamiadecimal string.
 
     """
-    cmd = f"openssl rand -hex {length}"
+    cmd = f"openssl rand -base64 {length}"
```

Comment:
This could cause issues since`-base64` can include characters like '+' or "=" that might not be accepted as account or bucket names: https://docs.aws.amazon.com/AmazonS3/latest/userguide/bucketnamingrules.html

Normally I'd suggest adding a `format` argument to the function and use that to determine whether to use `hex` or `base64`, but since the name of the function specifies `hex` I suggest you add this option under a separate function to avoid confusion.

Reply:
Thanks for pointing out this.
I will try to modify code in main framework to generate the required data